app/src/test/java/gov/va/vro/EvidenceSummaryDocumentProcessorTest.java

```diff
@@ -23,53 +22,52 @@
 
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
-import java.util.Date;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @ActiveProfiles("test")
 @Import(AppTestConfig.class)
 @CamelSpringBootTest
 public class EvidenceSummaryDocumentProcessorTest extends BaseControllerTest {
 
-  @Value("classpath:test-data/pdf-generator-input-01.json")
+  @Value("classpath:test-data/pdf-generator-mas.json")
   private Resource pdfGeneratorInput01;
 
   @Autowired protected ClaimRepository claimRepository;
 
-  @Test
```

Comment:
I could not figure out why this test was failing

Reply:
@dimitri-amida

Reply:
The PDF generator response indicates that the request was successful but the assertion on line 61 fails